scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2945,7 +2973,18 @@ void BPF_STRUCT_OPS(layered_update_idle, s32 cpu, bool idle)
 	 * see idle set or we see the task in one of the DSQs.
 	 */
 	llc_id = cpu_to_llc_id(cpu);
+
 	bpf_for(layer_id, 0, nr_layers) {
+		if((layer = lookup_layer(layer_id)) && 
+			(layer_cpumask = lookup_layer_cpumask(layer_id)) &&
+			!bpf_cpumask_test_cpu(cpu, layer_cpumask) &&
```

Comment:
@etsal correct me if I am wrong but this is all that's needed to have a similar effect on cpuset'ed workloads as the one wrt/ skip-rnode right (i.e. don't bother checking dsq's which cannot be processed)?